src/dll/edbgCipDll.C

```diff
@@ -34,13 +35,20 @@
 #include <ecmdReturnCodes.H>
 #include <ecmdDataBuffer.H>
 #include <ecmdSharedUtils.H>
+#include <ecmdChipTargetCompare.H>
```

Comment:
Why did this need to be added, I don't see anything in the code that requires it?